tests/schema_/test_embedded_model.py

```diff
@@ -552,3 +552,41 @@ def test_embedded_alter_field_ignored(self):
         new_field.set_attributes_from_name("new")
         with connection.schema_editor() as editor, self.assertNumQueries(0):
             editor.alter_field(Author, old_field, new_field)
+
+
+@skipUnlessDBFeature("gis_enabled")
+class GISTests(TestMixin, TransactionTestCase):
+    @isolate_apps("schema_")
+    def test_create_model(self):
+        """
+        Spatial indexes for embedded GIS fields are created when the collections are
+        created.
+        """
+        from django.contrib.gis.db.models import PointField  # noqa: PLC0415
+
+        class Place(EmbeddedModel):
+            name = models.CharField(max_length=10)
+            location = PointField()
+
+            class Meta:
+                app_label = "schema_"
+
+        class Author(models.Model):
+            birthplace = EmbeddedModelField(Place)
+
+            class Meta:
```

Comment:
Can we have a test that confirms field storage and database schema? For `MongoDB` GeoJSON format still requires:
`{ type: <GeoJSON Type>, ...}` as the data format. I see in the `Adapter` does the work to ensure proper schema conversion, but I think it still helps to have a test.

Reply:
Like a query with `pymongo.find()`? Another thing to consider (if it assuages your concern) is that MongoDB raises an error (e.g. "can't extract geo keys") if a spatial index receives invalid data.